geomesa-blobstore/README.md

```diff
@@ -17,3 +17,40 @@ The class `WKTFileHandler` serves as an example of that latter approach.
 
 `FileHandler` implementations are loaded at runtime by the Java ServiceLoader.  As such, available 
 implementations should have their classname registered in `META-INF/services/org.locationtech.geomesa.blob.core.handlers.FileHandler`.
+
+## REST api
+
+The RESTful api for the Blobstore can be easily utilized via cURL as so:
+
+Before we can do anything we must first register a blob store. This typically only needs to be run once in the lifetime
+of the webserver, so if the server is shutdown or restarted the registration call will need to be repeated.
```

Comment:
we don't pull it in directly - we actually do have an abstraction over it. We invoke some geoserver classes through spring beans to get the folder to persist things to. not sure if that is allowed or not... see e.g.
https://github.com/locationtech/geomesa/blob/master/geomesa-web/geomesa-web-data/src/main/resources/applicationContext.xml#L18-L49